examples/06_Combining_Models.ipynb

```diff
@@ -250,6 +250,247 @@
    "source": [
     "## Ensemble Model"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "An ensemble model is an approach to modeling where one or more different models are simulated together and then aggregated into a single prediction. This is generally done to improve the accuracy of prediction when you have multiple models that each represent part of the behavior or represent a distribution of different behaviors.\n",
```

Comment:
> when really we mean multiple instances of the same physical model but with different parameters (right?)

It can be that or different models (e.g., electrochemical and equiv circuit). But I guess what's missing here is that they are of the same system. Let me take a stab at clearing this up